themes-default/slim/src/components/quality-pill.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+    <span :title="title" :class="pill.class">{{ pill.text }}</span>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+
+/**
```

Comment:
Change from this
```
/**
 * @typedef {Object} Quality
```

to this

```
/**
 * Quality pill
 *
 * @typedef {Object} Quality
```

Reply:
It doesn't represent the quality pill, it represents a split quality:
`{ allowed: [ 1, 2, 4 ], preferred: [ 128, 256 ] }`

Reply:
How's that?
```js
/**
 * An object representing a split quality.
 * @typedef {Object} Quality
 * @property {number[]} allowed - Allowed qualities
 * @property {number[]} preferred - Preferred qualities
 */
```